rest-api/api/participant_summary_api.py

```diff
@@ -10,7 +10,10 @@
 
 class ParticipantSummaryApi(BaseApi):
   def __init__(self):
-    super(ParticipantSummaryApi, self).__init__(ParticipantSummaryDao())
+    super(ParticipantSummaryApi, self).__init__(
+      ParticipantSummaryDao(),
+      get_returns_children=True
```

Comment:
I have a performance concern when users use this API for sync'ing records.  I think we should put this under a new temporary testing API, excluded in Prod, to compare various usage scenarios vs the current API.